src/components/button-toggle/button-toggle.spec.js

```diff
@@ -1,34 +1,27 @@
 import React from 'react';
 import { mount } from 'enzyme';
+import { ThemeProvider } from 'styled-components';
 import TestRenderer from 'react-test-renderer';
 import 'jest-styled-components';
 import guid from '../../utils/helpers/guid';
-import classicTheme from '../../style/themes/classic';
-import smallTheme from '../../style/themes/small';
+import ClassicTheme from '../../style/themes/classic';
+import SmallTheme from '../../style/themes/small';
 import ButtonToggle from './button-toggle.component';
 import { assertStyleMatch } from '../../__spec_helper__/test-utils';
 import { StyledButtonToggleIcon } from './button-toggle.style';
 
 jest.mock('../../utils/helpers/guid');
 guid.mockImplementation(() => 'guid-12345');
 
-const testRender = (props) => {
-  return TestRenderer.create(<ButtonToggle { ...props }>Button</ButtonToggle>);
-};
-
-const render = (props) => {
-  return mount(<ButtonToggle { ...props }>Button</ButtonToggle>);
-};
-
 describe('ButtonToggle', () => {
   describe('Classic theme', () => {
     it('renders correctly with default settings', () => {
-      const wrapper = testRender({ theme: classicTheme });
+      const wrapper = renderWithTheme({ theme: ClassicTheme }, TestRenderer.create);
       expect(wrapper).toMatchSnapshot();
     });
     it('renders correctly with large buttonIcon and large size', () => {
-      const wrapper = render({
-        theme: classicTheme,
+      const wrapper = renderWithTheme({
```

Comment:
What do you think to create beforeEach in the case when we have same props provided?

Reply:
that's a good idea, but we don't have the same prop sets here